### ISSUE TITLE
Unhandled exception in `is` error

### ISSUE THREAD

**Issue by mcraftpete** [rlf/uSkyBlock#368]
Hi,
I occasionally get these errors ...

```
[12:15:27] [Server thread/INFO]: Jaydlake issued server command: /is
[12:15:27] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'is' in plugin uSkyBlock v2.3-HF3
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot.jar:git-Spigot-8a983f9-e81edfc]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot.jar:git-Spigot-8a983f9-e81edfc]
	at org.bukkit.craftbukkit.v1_8_R1.CraftServer.dispatchCommand(CraftServer.java:646) ~[spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.PlayerConnection.handleCommand(PlayerConnection.java:1115) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.PlayerConnection.a(PlayerConnection.java:950) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.PacketPlayInChat.a(PacketPlayInChat.java:26) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.PacketPlayInChat.a(PacketPlayInChat.java:53) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.PacketHandleTask.run(SourceFile:13) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_05]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_05]
	at net.minecraft.server.v1_8_R1.MinecraftServer.z(MinecraftServer.java:696) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.DedicatedServer.z(DedicatedServer.java:316) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.MinecraftServer.y(MinecraftServer.java:634) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.MinecraftServer.run(MinecraftServer.java:537) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_05]
Caused by: java.lang.NullPointerException
	at us.talabrek.ultimateskyblock.handler.VaultHandler.checkPerk(VaultHandler.java:41) ~[?:?]
	at us.talabrek.ultimateskyblock.SkyBlockMenu.addMainMenu(SkyBlockMenu.java:693) ~[?:?]
	at us.talabrek.ultimateskyblock.SkyBlockMenu.displayIslandGUI(SkyBlockMenu.java:567) ~[?:?]
	at us.talabrek.ultimateskyblock.command.IslandCommand.onCommand(IslandCommand.java:61) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot.jar:git-Spigot-8a983f9-e81edfc]
	... 14 more
```

**Comment by rlf** [comment]
Sounds extremely weird - since that would mean the `perms` object is NULL, and that should only ever happen, if someone enters the island command during bootup/startup.

**Comment by mcraftpete** [comment]
I am getting more of this same error .. it actually crashes the server each time...
12:51:28] [Server thread/INFO]: Franyy issued server command: /is w Franyy
[12:51:28] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'is' in plugin uSkyBlock v2.3-HF3
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot.jar:git-Spigot-8a983f9-e81edfc]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot.jar:git-Spigot-8a983f9-e81edfc]
	at org.bukkit.craftbukkit.v1_8_R1.CraftServer.dispatchCommand(CraftServer.java:646) ~[spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.PlayerConnection.handleCommand(PlayerConnection.java:1115) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.PlayerConnection.a(PlayerConnection.java:950) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.PacketPlayInChat.a(PacketPlayInChat.java:26) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.PacketPlayInChat.a(PacketPlayInChat.java:53) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.PacketHandleTask.run(SourceFile:13) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_05]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_05]
	at net.minecraft.server.v1_8_R1.MinecraftServer.z(MinecraftServer.java:696) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.DedicatedServer.z(DedicatedServer.java:316) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.MinecraftServer.y(MinecraftServer.java:634) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at net.minecraft.server.v1_8_R1.MinecraftServer.run(MinecraftServer.java:537) [spigot.jar:git-Spigot-8a983f9-e81edfc]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_05]
Caused by: java.lang.NullPointerException
	at us.talabrek.ultimateskyblock.handler.VaultHandler.checkPerk(VaultHandler.java:41) ~[?:?]
	at us.talabrek.ultimateskyblock.command.island.WarpCommand.doExecute(WarpCommand.java:44) ~[?:?]
	at us.talabrek.ultimateskyblock.command.island.RequirePlayerCommand.execute(RequirePlayerCommand.java:33) ~[?:?]
	at us.talabrek.ultimateskyblock.command.common.CompositeUSBCommand.execute(CompositeUSBCommand.java:116) ~[?:?]
	at us.talabrek.ultimateskyblock.command.common.AbstractCommandExecutor.onCommand(AbstractCommandExecutor.java:21) ~[?:?]
	at us.talabrek.ultimateskyblock.command.IslandCommand.onCommand(IslandCommand.java:64) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot.jar:git-Spigot-8a983f9-e81edfc]
	... 14 more

**Comment by rlf** [comment]
What version of Vault are you using?
And please provide the server-log during startup (pastebin)

**Comment by mcraftpete** [comment]
http://pastebin.com/iLDfJiaZ
Hope this has everything you need :) , I do still get these errors and they do cause server crashes so any infor would be great.

**Comment by rlf** [comment]
Thanks for the log Pete, but, it doesn't show what I need.
Can I get the server log from startup, till the error occurred? Or is that too big?

**Comment by mcraftpete** [comment]
ld be really big , as its hard to know when it will happen , its not like it happens all the time but 3 or 4 times a day it does and crashes the server, its same error each time , its like when they join and they haven't played fro a little whil it has no region created and this seems to be the main issue , no region causing the other issues.
Just now I had a player login with same error , and had no region at his island , reclogged and fregion was there.

**Comment by rlf** [comment]
Should be fixed in v2.4.1 - this error happens, when players log in too fast after restart, i.e. before the Vault permission system has been initialized.
